config/locales/teacher_interface.en.yml

```diff
@@ -320,7 +320,7 @@ en:
         teacher_interface/work_history_contact_form:
           attributes:
             contact_name:
-              blank: Enter the full name
+              blank: Enter this person's full name
```

Comment:
We should change "Enter the email address" below to "Enter this person’s email address"